client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/index.js

```diff
@@ -159,11 +159,25 @@ const RatesStep = props => {
 	const updateRateHandler = ( packageId, serviceId, signatureRequired ) =>
 		props.updateRate( orderId, siteId, packageId, serviceId, signatureRequired );
 
+	// Preselect rates for packages that have only one rate available.
+	forEach( form.packages.selected, ( selectedRate, pckgId ) => {
+		// Skip preselection for already selected values.
+		if( values[ pckgId ] !== "" ) {
```

Comment:
I think we're generally using 'yoda' conditional check orders in this codebase 🙄, so the order should be switched

Reply:
![image](https://user-images.githubusercontent.com/17271089/68966568-bc35d480-07de-11ea-878d-1223a2a23348.png)